modules/logging-enabling-loki-alerts.adoc

```diff
@@ -1,31 +1,27 @@
 // Module included in the following assemblies:
-// logging-5-7-configuration
+//
+// logging/logging_alerts/custom-logging-alerts.adoc
 
 :_content-type: PROCEDURE
-[id="logging-loki-alerts_{context}"]
-= Enabling log based alerts with Loki
-Loki alerting rules use link:https://grafana.com/docs/loki/latest/logql/[LogQL] and follow link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus formatting]. You can set log based alerts by creating an `AlertingRule` custom resource (CR). `AlertingRule` CRs may be created for `application`, `audit`, or `infrastructure` tenants.
+[id="logging-enabling-loki-alerts_{context}"]
+= Enabling log-based alerts with Loki
```

Comment:
Feels more like this no? 
```suggestion
= Creating a new log-based alerting rule
```

Reply:
went with "Creating a log-based alert with Loki" so that it's clear from the title that you need Loki to do this